.vib/alertmanager/goss/vars.yaml

```diff
@@ -0,0 +1,10 @@
+binaries:
+  - alertmanager
+  - amtool
+files:
+  - paths:
+      - /opt/bitnami/alertmanager/conf/config.yml
```

Comment:
You are already checking this file in the `check-app-run` test, so there is no need to double check it here.